runtime/bindings/python/tests/test_ngraph/test_basic.py

```diff
@@ -361,6 +363,67 @@ def test_node_output():
 
     assert [output0.get_index(), output1.get_index(), output2.get_index()] == [0, 1, 2]
 
+def test_node_input_size():
+    node = ops.add([1], [2])
+    assert node.get_input_size() == 2
+
+def test_node_input_values():
+    shapes = [Shape([3]), Shape([3])]
+    data1 = np.array([1, 2, 3])
+    data2 = np.array([3, 2, 1])
+
+    node = ops.add(data1, data2)
+
+    assert node.get_input_size() == 2
+
+    assert np.equal(
+        [input_node.get_shape() for input_node in node.input_values()],
+        shapes
+    ).all()
+
+    assert np.equal(
+        [node.input_value(i).get_shape() for i in range(node.get_input_size())],
+        shapes
+    ).all()
+
+    assert np.allclose(
+        [input_node.get_node().get_vector() for input_node in node.input_values()],
+        [data1, data2]
+    )
+
+    assert np.allclose(
+        [node.input_value(i).get_node().get_vector() for i in range(node.get_input_size())],
+        [data1, data2]
+    )
+
+def test_node_input_tensor():
+    data1 = np.array([1, 2, 3])
+    data2 = np.array([3, 2, 1])
+
+    node = ops.add(data1, data2)
+
+    assert(isinstance(node.get_input_tensor(0), descriptor.Tensor))
```

Comment:
Can you first assign it somewhere and then assert? Just curious about behavior.

Reply:
Sure, done.